tests/typecheck/fields/test_related.yml

```diff
@@ -600,7 +608,7 @@
                     publisher = models.ForeignKey(Publisher, on_delete=models.CASCADE)
                 class Author(models.Model):
                     blogs = models.ManyToManyField(Blog)
-                    file = models.FileField()
+                    file = FileField()
```

Comment:
Why is it changed?

Reply:
I was fiddling around with some stuff in the beginning. This should be reverted.